src/AggregateFunctions/AggregateFunctionTopK.cpp

```diff
@@ -67,36 +68,36 @@ class AggregateFunctionTopKIPv4 : public AggregateFunctionTopK<DataTypeIPv4::Fie
 public:
     using AggregateFunctionTopK<DataTypeIPv4::FieldType, is_weighted>::AggregateFunctionTopK;
 
-    AggregateFunctionTopKIPv4(UInt64 threshold_, UInt64 load_factor, const DataTypes & argument_types_, const Array & params)
+    AggregateFunctionTopKIPv4(UInt64 threshold_, UInt64 load_factor, UInt8 include_statistics_, const DataTypes & argument_types_, const Array & params)
         : AggregateFunctionTopK<DataTypeIPv4::FieldType, is_weighted>(
             threshold_,
             load_factor,
+            include_statistics_,
             argument_types_,
-            params,
-            std::make_shared<DataTypeArray>(std::make_shared<DataTypeIPv4>()))
+            params)
     {}
 };
 
 
 template <bool is_weighted>
-IAggregateFunction * createWithExtraTypes(const DataTypes & argument_types, UInt64 threshold, UInt64 load_factor, const Array & params)
+IAggregateFunction * createWithExtraTypes(const DataTypes & argument_types, UInt64 threshold, UInt64 load_factor, UInt8 include_statistics, const Array & params)
```

Comment:
Why include_statistics is UInt8 and not a bool?

Reply:
Yes, will change to bool as well.